docs/topics/signalen-and-standard-for-public-code.md

```diff
@@ -85,64 +87,64 @@ You SHOULD prefer file formats that can easily be version controlled. | yes | Gi
-Any translation MUST be up to date with the English version and vice versa. | | See above
-There SHOULD be no acronyms, abbreviations, puns or legal/domain specific terms in the codebase without an explanation preceding it or a link to an explanation. | | Glossary needs expanding.
-The name of the project or codebase SHOULD be descriptive and free from acronyms, abbreviations, puns or branding. | yes | SIA --> Signalen
-Documentation SHOULD aim for a lower secondary education reading level, as recommended by the Web Content Accessibility Guidelines. | | Tech documentation will be harder, but a check on signalen.org is needed.
-Any code, documentation and tests MAY have a translation. | |
+All codebase documentation MUST be in English. | no | If signalen.org is the source for some documentation (and not only a marketing asset), then it should be also available in English.
+All code MUST be in English, except where policy is machine interpreted as code. | no | Some workflow contains Dutch states without English translation
+Any translation MUST be up to date with the English version and vice versa. |  | See above
+There SHOULD be no acronyms, abbreviations, puns or legal/non-English/domain specific terms in the codebase without an explanation preceding it or a link to an explanation. |  | Glossary needs expanding.
+The name of the codebase SHOULD be descriptive and free from acronyms, abbreviations, puns or organizational branding. | yes | SIA --> Signalen
+Documentation SHOULD aim for a lower secondary education reading level, as recommended by the [Web Content Accessibility Guidelines 2](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=315#readable). |  | Tech documentation will be harder, but a check on signalen.org is needed.
+Any code, documentation and tests MAY have a translation. |  |
 
 ## [Use open standards](https://standard.publiccode.net/criteria/open-standards.html)
 
 - [ ] compliant with this criterion.
 
 Requirement | meets | links and notes
 -----|-----|-----
-For features of a codebase that facilitate the exchange of data the codebase MUST use an open standard that meets the Open Source Initiative Open Standard Requirements. | | [StUF-ZKN](https://www.gemmaonline.nl/index.php/Sectormodel_Zaken:_StUF-ZKN) OpenAPI2 spec, GraphQL, Oauth2, OpenID Connect, SMTP, JSON?
-If no existing open standard is available, effort SHOULD be put into developing one. |  | Maybe we need a standard way to forward the complaint between bodies, e.g.: City <--> Provice?
+For features of a codebase that facilitate the exchange of data the codebase MUST use an open standard that meets the Open Source Initiative Open Standard Requirements. |  | [StUF-ZKN](https://www.gemmaonline.nl/index.php/Sectormodel_Zaken:_StUF-ZKN) OpenAPI2 spec, GraphQL, Oauth2, OpenID Connect, SMTP, JSON?
```

Comment:
I don't know if the list of open standards is meant to be a description of Signalen or just examples. If it is the former we use `openapi: 3.0.0` and have removed any support for GraphQL (it was proof of concept, not being used, and a source of extra maintenance).